app.html

```diff
@@ -18,5 +26,9 @@
   <script src="graph_plot.js"></script>
   <script src="gradient_descent.js"></script>
   <script src="test_graph_plot.js"></script>
+  <div id="documentation" align="center">
+    <p style="color:black;font-size:50px;"> Gradient Descent Algorithm</p>
```

Comment:
Just a personal preference but I think it is better if at least the title is above the visualisation (the description can stay below). Your call :)